frontend/packages/insights-plugin/src/components/InsightsPopup/index.tsx

```diff
@@ -36,7 +36,7 @@ export const InsightsPopup: React.FC<PrometheusHealthPopupProps> = ({ responses,
                 y: v,
               }))}
               title={`${numberOfIssues}`}
-              subTitle="Total issues"
+              subTitle={`Total ${numberOfIssues > 1 ? 'issues' : 'issue'}`}
```

Comment:
```suggestion
              subTitle={`Total ${numberOfIssues === 1 ? 'issue' : 'issues'}`}
```

This is better in general since you want "0 issues" if it's ever 0. I realize that `hasIssues` should be false here, but I'd rather code defensively in case that ever changes.

Reply:
Done